app/code/Magento/Bundle/Test/Unit/Model/LinkManagementTest.php

```diff
@@ -686,9 +701,10 @@ public function testSaveChild()
         $linkProductId = 45;
         $parentProductId = 32;
         $bundleProductSku = 'bundleProductSku';
+        $websiteId = 100;
 
         $productLink = $this->getMockBuilder(LinkInterface::class)
-            ->setMethods(['getSku', 'getOptionId', 'getSelectionId'])
+            ->setMethods(['getSku', 'getOptionId', 'getSelectionId', 'getWebSiteId'])
```

Comment:
This method doesn't exist anymore on the Link, please remove it.

Reply:
Done